spec/features/messaging_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.feature 'Messaging' do
+  let!(:alice) { create(:user, name: 'Alice') }
+  let!(:bob) { create(:user, name: 'Bob') }
+
+  scenario 'a user sends a messages' do
+    sign_in(alice)
+
+    click_on('My Account')
+    click_on('Messages')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.